platforms/chibios/drivers/vendor/RP/RP2040/ps2_vendor.c

```diff
@@ -22,7 +22,7 @@
 #endif
 
 #if PS2_DATA_PIN + 1 != PS2_CLOCK_PIN
-#    error PS/2 Clock pin must be followed by data pin!
+#    error PS/2 data pin must be followed by clock pin!
```

Comment:
The language in this error messages has caused confusion for quite a few users, there was an attempt to correct it in #21786. 

I think the check is still clearer than the error it's self. So maybe something along the lines of this would be better?

```suggestion
#    error PS/2 clock pin must be data pin + 1!
```

Reply:
Yes I agree it is clearer this way